docs/yourbase-ruby.md

```diff
@@ -0,0 +1,59 @@
+# YourBase Ruby Acceleration
+
+Tests are important. For large monoliths, they're also a major source of drag on velocity.
+
+YourBase is a tool that traces your tests to determine which functions each test depends on. It can later use this information to determine which tests do not need to run because their code paths have not changed. These tests are skipped automatically.
+
+YourBase works with Ruby versions >= 2.3
+
+## Getting Started
+No configuration, setup, or babysitting required. To get started, you need the YourBase gem and a download token.  To request a token, please visit [YourBase.io](https://yourbase.io/download)
+
+Once you have a token, simply follow the steps below:
+```sh
+YOURBASE_DOWNLOAD_TOKEN=<enter token> 
+bundle add yourbase-rspec --git "https://$YOURBASE_DOWNLOAD_TOKEN:x-oauth-basic@github.com/yourbase/yourbase-rspec-skipper-engine.git"
+
+bundle install
```

Comment:
I recommend following the advice in https://www.vidarholen.net/contents/blog/?p=958:

```suggestion
bundle add yourbase-rspec --git "https://${YOURBASE_DOWNLOAD_TOKEN?}:x-oauth-basic@github.com/yourbase/yourbase-rspec-skipper-engine.git" &&
bundle install
```

Reply:
I like this!  Very clear.